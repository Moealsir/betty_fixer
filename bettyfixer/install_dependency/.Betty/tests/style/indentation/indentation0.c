int func0(void)
{
    return (0);
}

int func1(void)
{
	return (0);
}
