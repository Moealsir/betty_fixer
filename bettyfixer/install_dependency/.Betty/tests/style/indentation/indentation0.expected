tests/style/indentation/indentation0.c:3: WARNING: please, no spaces at the start of a line
total: 0 errors, 1 warnings, 9 lines checked
